sections/collapsible-content.liquid

```diff
@@ -0,0 +1,382 @@
+{{ 'component-accordion.css' | asset_url | stylesheet_tag }}
+{{ 'collapsible-content.css' | asset_url | stylesheet_tag }}
+
+<div class="collapsible-content{% if section.settings.layout == 'no_container' %} color color-{{ section.settings.color_scheme }}{% endif %}{% unless section.settings.layout == 'no_container' %} color-{{ section.settings.container_color_scheme }}{% endunless %}{% if section.settings.layout == 'section_container' %} page-width{% endif %}">
+  <div class="collapsible-content-wrapper{% if section.settings.layout == 'section_container' %} color color-{{ section.settings.color_scheme }}{% endif %}">
+    <div class="{% if section.settings.image == blank %} collapsible-content-wrapper--narrow{% else %} page-width{% endif%}">
+      <div style="text-align: {{ section.settings.heading_alignment }}">
+        {%- if section.settings.caption != blank -%}
+          <p class="caption-with-letter-spacing"> {{ section.settings.caption }}</p>
+        {% endif %}
+        {%- if section.settings.heading != blank -%}
+          <h2 class="h1">{{ section.settings.heading }}</h2>
+        {% endif %}
+      </div>
+      <div class="grid grid--1-col grid--2-col-tablet collapsible-content__grid{% if section.settings.desktop_layout == 'image_first' %} collapsible-content__grid--reverse{% endif %}">
+        <div class="grid__item grid__item--content">
+          {%- for block in section.blocks -%}
+            <div class="accordion{% if section.settings.layout == 'row_container' %} color color-{{ section.settings.color_scheme }}{% endif %}" {{ block.shopify_attributes }}>
+              <details id="Details-{{ block.id }}-{{ section.id }}"{% if section.settings.open_first_collapsible_row and forloop.first %} open{% endif %}>
+                <summary>
+                  <div class="summary__title">
+                    {% render 'icon-accordion', icon: block.settings.icon %}
+                    <h2 class="h4 accordion__title">
+                      {{ block.settings.heading | default: block.settings.page.title }}
+                    </h2>
+                  </div>
+                  {% render 'icon-caret' %}
+                </summary>
+                <div class="accordion__content rte" id="ProductAccordion-{{ block.id }}-{{ section.id }}">
+                  {{ block.settings.row_content }}
+                  {{ block.settings.page.content }}
+                </div>
+              </details>
+            </div>
+          {%- endfor -%}
+        </div>
+        {%- if section.settings.image != blank -%}
+          <div class="grid__item">
```

Comment:
Since the image is visually first on mobile, and `Image first` is an option, could we swap the order of the `.grid__item` elements? This will generate a more natural content flow and match the visual order when navigating by screen reader/braille reader.